deploy/helm/flightctl/charts/keycloak/templates/keycloak-realm-secret.yaml

```diff
@@ -619,34 +619,34 @@ stringData:
           "clientId": "flightctl",
           "name": "flightctl",
           "description": "",
-          {{- if .Values.route.hostname }}
-          "rootUrl": "https://{{ .Values.route.hostname }}"",
-          "adminUrl": "https://{{ .Values.route.hostname }}"",
-          "baseUrl": "https://{{ .Values.route.hostname }}"",
-          {{- else }}
           "rootUrl": "https://auth.{{ include "flightctl.getBaseDomain" . }}",
           "adminUrl": "https://auth.{{ include "flightctl.getBaseDomain" . }}",
           "baseUrl": "https://auth.{{ include "flightctl.getBaseDomain" . }}",
-          {{- end }}
           "surrogateAuthRequired": false,
           "enabled": true,
           "alwaysDisplayInConsole": false,
           "clientAuthenticatorType": "client-secret",
           {{- if .Values.realm.redirectUris }}
           "redirectUris": {{ toJson .Values.realm.redirectUris }},
+          {{- else if eq (include "flightctl.getServiceExposeMethod" .) "nodePort" }}
+          "redirectUris": [
+            "http://127.0.0.1/callback",
+            "{{ include "flightctl.getUIHttpScheme" . }}://{{ include "flightctl.getBaseDomain" . }}",
```

Comment:
woulnd't we need here the target nodeport?

Reply:
ports are ignored in redirectURIs